acceptance/acceptance_test.go

```diff
@@ -137,11 +142,18 @@ func TestAcceptance(t *testing.T) {
 		h.AssertNil(t, err)
 	}
 
+	tmpPreviousPackFixturesPath, err := ioutil.TempDir("", "previous-pack-fixtures")
+	h.AssertNil(t, err)
+	defer os.RemoveAll(tmpPreviousPackFixturesPath)
+
+	h.RecursiveCopy(t, previousPackFixturesPath, tmpPreviousPackFixturesPath)
+	h.RecursiveCopy(t, filepath.Join("testdata", "pack_previous_fixtures_overrides"), tmpPreviousPackFixturesPath)
```

Comment:
I _think_ I understand why this is needed. Maybe it's worth creating a `README.md` in that directory that entails exactly _why_ you would put stuff in that directory.

Also, would it not suffer from the same problems we originally had with "pack_previous" in which you had to update the repo after a release because "current" was (de)moted to "previous"?

Reply:
Would vote on punting on the docs so we can get this out. Thoughts @jromero?

It won't suffer from the same problems if it just has the version-related files like it has now. But true, it could suffer in general. Hopefully these are the only type of files we'd need. We'll keep an eye on this.